src/reconcile/rec_col.c

```diff
@@ -1404,17 +1395,14 @@ __wt_rec_col_var(
                      * copy; read it into memory.
                      */
                     WT_ERR(__wt_dsk_cell_data_ref_kv(session, WT_PAGE_COL_VAR, vpack, orig));
-
-                    ovfl_state = OVFL_IGNORE;
```

Comment:
why this statement is removed?

Reply:
Oops - I double checked after your comment. I'll put it back.

I originally didn't think it's necessary - as we iterate over entries in the in-memory page, it's set to `OVFL_IGNORE` each time so we don't hit the `UNUSED` or `USED` cases. And the inner loop for repeated records doesn't use the `OVFL_IGNORE` state except for this switch statement, where I didn't think it was the right thing to reset it.

But I see now that it helps to skip the unpacking step when a run of repeated records goes from overflow to non-overflow.

Reply:
Fixed in 57b1879